en/news/_posts/2019-04-17-ruby-2-6-3-released.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: news_post
+title: "Ruby 2.6.3 Released"
+author: "naruse"
+translator:
+date: 2019-04-017 00:00:00 +0000
+lang: en
+---
+
+Ruby 2.6.3 has been released.
+
+This release adds support for New Japanese Era "令和" (Reiwa).
+It updates the Unicode version to 12.1 beta ([#15195](https://bugs.ruby-lang.org/issues/15195), and updates date library ([#15742](https://bugs.ruby-lang.org/issues/15742).
```

Comment:
```suggestion
It updates the Unicode version to 12.1 beta ([#15195](https://bugs.ruby-lang.org/issues/15195)), and updates date library ([#15742](https://bugs.ruby-lang.org/issues/15742)).
```

It updates the Unicode version to 12.1 beta ([#15195](https://bugs.ruby-lang.org/issues/15195)), and updates date library ([#15742](https://bugs.ruby-lang.org/issues/15742)).